src/Command/Support.js

```diff
@@ -124,8 +124,9 @@ module.exports = Command.extend({
             })
             .catch(() => {
                 /* If something went wrong with room creation, we delete it (if it was created) and notify the user */
-                supportChannel.delete().catch(() => {
-                });
+                if (supportChannel) {
```

Comment:
This threw an error due to a misconfiguration on my side, where a channel was never created to begin with, so it couldn't be deleted. Figured I might as well address this whilst we are at it.